app/services/course_search_service.rb

```diff
@@ -39,9 +39,15 @@ def call
       outer_scope = outer_scope.descending_canonical_order
       outer_scope = outer_scope.select("provider.provider_name", "course.*")
     elsif sort_by_distance?
-      outer_scope = outer_scope.joins(sites_with_distance_from_origin)
-      outer_scope = outer_scope.select("course.*, distance")
-      outer_scope = outer_scope.order(:distance)
+      outer_scope = outer_scope.joins(courses_with_distance_from_origin)
+      boosted_distance_column = "
```

Comment:
Would `distance_with_university_area_adjustment` or something like that be clearer. Bit of a mouthful admittedly 😬